tests/python/test_scan.py

```diff
@@ -4,6 +4,10 @@
 
 @test_utils.test(arch=[ti.cuda, ti.vulkan], exclude=[(ti.vulkan, "Darwin")])
 def test_scan():
+
+    # A global prefix sum wrapper, only need to be initialized once.
+    executor = ti.algorithms.PrefixSumExecutor()
```

Comment:
nit: shall we use `ti.alg.` so that it's shorter for users? :P

Reply:
IMHO, `algorithms` makes it easier to be distinguished from `linalg`. `algorithms` means element-manipulating algorithms (like C++ algorithm library) and the `linalg` is for mathematic linear algebra.

Reply:
aha I forgot that we have linalg, then all good!